src/unittests/floating_point.spec.ts

```diff
@@ -2714,12 +2721,12 @@ const kCosIntervalThirdPiCases = {
     // cos(-1.046875) = 0.50027931
     {
       input: kValue.f16.negative.pi.third,
-      expected: FP['f16'].correctlyRoundedInterval(0.50027931).bounds(),
+      expected: FP['f16'].correctlyRoundedInterval(0.50027931).endpoints(),
```

Comment:
Just a note (no action required), I am thinking that can we make the expectation also take an interval as well as single point and endpoints (i.e. array of point)...